FordLabs/retroquest
# Give archives pages individual routes

## Overview
This PR added separate routes for all archived pages (thoughts list, thoughts, and action items)

### Demo
Archived Thoughts List Page:
<img width="1336" alt="Screen Shot 2022-11-10 at 4 28 35 PM" src="https://user-images.githubusercontent.com/17144844/201209948-f000fa50-50bc-4ade-9acc-b1cf093b0e64.png">

Archived Thoughts Page:
<img width="1278" alt="Screen Shot 2022-11-10 at 4 28 49 PM" src="https://user-images.githubusercontent.com/17144844/201209957-3973ccfd-8a22-4cdb-8358-41be9f3d279b.png">

Archived Action Items Page:
<img width="1426" alt="Screen Shot 2022-11-10 at 4 28 58 PM" src="https://user-images.githubusercontent.com/17144844/201209967-c9255957-f317-459a-abf5-7c3027a9340a.png">


## Testing Instructions
1) Archive several board of thoughts and closed action items.
2) Go to the archived page
3) Ensure clicking on "Archived" takes you to `/team/{teamId}/archives/thoughts`
4) Ensure "Thoughts" has an underline under it showing it as the active page
5) Ensure everything renders properly on the thoughts page
6) Click on "View" on one of the archived boards.
7) Ensure it shows the archived board page at `/team/{teamId}/archives/thoughts/{boardId}` and renders everything properly. Refresh the page and ensure the page still loads properly.
8) Click "Action Items" and ensure the route changes to `/team/{teamId}/archives/action-items`. Ensure everything renders properly.
9) Refresh the action items page and ensure everything still renders properly.
